imports/plugins/core/router/lib/router.js

```diff
@@ -313,15 +313,13 @@ Router.reload = () => {
  */
 Router.isActiveClassName = (routeName) => {
   const current = Router.current();
-  const group = current.route.group;
-  let prefix = "";
+  const { group } = current.route || {};
+  const defaultPrefix = "";
 
   if (current.route) {
-    const path = current.route.path;
-
-    if (group && group.prefix) {
-      prefix = current.route.group.prefix;
-    }
+    const { path } = current.route;
+    const { prefixArg } = group || {};
```

Comment:
Where does `prefixArg` come from? It looks like previously it was `current.route.group.prefix`